src/main/java/org/folio/eusage/reports/api/UseOverTimeApi.java

```diff
@@ -0,0 +1,77 @@
+package org.folio.eusage.reports.api;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.client.WebClient;
+import io.vertx.ext.web.openapi.RouterBuilder;
+import io.vertx.sqlclient.Tuple;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.tlib.RouterCreator;
+import org.folio.tlib.postgres.TenantPgPool;
+import org.folio.tlib.util.TenantUtil;
+
+public class UseOverTimeApi implements RouterCreator {
+  private static final Logger log = LogManager.getLogger(UseOverTimeApi.class);
+
+  static String titleEntriesTable(TenantPgPool pool) {
+    return pool.getSchema() + ".title_entries";
+  }
+
+  static String packageEntriesTable(TenantPgPool pool) {
+    return pool.getSchema() + ".package_entries";
+  }
+
+  static String titleDataTable(TenantPgPool pool) {
+    return pool.getSchema() + ".title_data";
+  }
```

Comment:
Separate implementation but, for now, including same utilities as the other one. That's not ideal, so something has to move to a common place.

Reply:
UseOverTimeApi.java merged into EusageReportsApi.java, allows to use the utility methods.